docs/source/testing/landing-page.mdx

```diff
@@ -0,0 +1,114 @@
+---
+title: Setting an Apollo Server landing page
+sidebar_title: Server landing page
+---
+
+> This feature is new in Apollo Server 3.
+
+You can configure the landing page that's served from Apollo Server's base URL. Aside from serving the default landing page, you can serve [GraphQL Playground](#graphql-playground) (an open-source GraphQL IDE), a completely [custom landing page](#custom-landing-page), or you can [disable the landing page entirely](#disabling-the-landing-page).
+
+## Default landing page
+
+Apollo Server serves the following landing page by default:
+
+<img class="screenshot" src="../images/as-landing-page.jpg" alt="Apollo Server default landing page" width="350"/>
```

Comment:
Well, this is the one it shows by default if NODE_ENV is not `production`. (ie ApolloServerPluginLandingPageLocalDefault). Probably should also show the production version too?